<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift Light</family>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="headBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="2" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>华文隶书</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Port</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <family>华文隶书</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Name</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEditAddress">
                 <property name="font">
                  <font>
                   <family>华文楷体</family>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEditPort">
                 <property name="font">
                  <font>
                   <family>华文隶书</family>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QLineEdit" name="lineEditName">
                 <property name="font">
                  <font>
                   <family>华文隶书</family>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>华文隶书</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Address</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="backgroundBtn">
               <property name="font">
                <font>
                 <family>Agency FB</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>background</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="connectBtn">
               <property name="font">
                <font>
                 <family>Agency FB</family>
                 <pointsize>14</pointsize>
                </font>
               </property>
               <property name="text">
                <string>connect</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <family>华文隶书</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>聊天记录</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="clientState">
               <property name="font">
                <font>
                 <family>华文隶书</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="text">
                <string>在线状态：离线</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="chatRecord">
             <property name="font">
              <font>
               <family>SimSun-ExtB</family>
               <pointsize>9</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEditSend"/>
         </item>
         <item>
          <widget class="QPushButton" name="sendBtn">
           <property name="font">
            <font>
             <family>华文隶书</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>send</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
